samples/eShopLite/AppHost/aspire-manifest.json

```diff
@@ -1,5 +1,22 @@
 {
   "resources": {
+    "nodeapp": {
+      "type": "dockerfile.v0",
+      "path": "..\\nodeapp\\Dockerfile",
+      "context": "..\\nodeapp",
+      "env": {
+        "NODE_ENV": "development",
+        "PORT": "{bindings.http.port}"
+      },
+      "bindings": {
+        "http": {
+          "scheme": "http",
+          "protocol": "tcp",
+          "transport": "http",
+          "containerPort": 5555
```

Comment:
Not sure we should be using this port, but we do need a port. This is the host port today, not the port the container is listening on (which happens to be passed in today).

Reply:
Ah the new overload tricked me!